src/types/rooms.ts

```diff
@@ -2,6 +2,7 @@ import { SoundConfigReference } from "./sounds";
 
 export enum RoomTypes {
   unclickable = "UNCLICKABLE",
+  iframe = "IFRAME",
 }
```

Comment:
Between this PR and https://github.com/sparkletown/sparkle/pull/1370, I feel like the naming of the `RoomTypes` might get a little confusing.

In particular, this `iframe` relates to changing how the room is displayed on the map itself (`MapRoom`), whereas in https://github.com/sparkletown/sparkle/pull/1370 we are using `video` to change how the `RoomModal` popup is shown.

I wonder if we can build a pattern into our naming choices here that distinguish between 'different thing on the map (`MapRoom`)' versus 'different thing in the modal (`RoomModal`)' in a clean/simple way?

Maybe it's as simple as using `videoModal` or similar for https://github.com/sparkletown/sparkle/pull/1370's type? What do you think?

(cc // @mike-lvov @sofisparkle in case you have any thoughts/ideas)

Reply:
Not sure how to best address this point, but happy for any guidance

Reply:
I've just started a thread on this in our internal tech channel ([ref](https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1621918122122300) 🔒) so will hopefully be able to give you some guidance one way or another soon.

Reply:
Any suggestions for how to address this one? Otherwise seems we're more or less ready to go with this feature

Reply:
> In particular, this iframe relates to changing how the room is displayed on the map itself (MapRoom)

Same with `unclickable`? Seems like it's #1370 that needs to be changed, not `RoomTypes`.

Since, 
> Maybe it's as simple as using videoModal or similar for #1370 type? What do you think?

is related to #1370, resolving this comment

Reply:
@mike-lvov cant remember where but there was discussion around this on slack, I know @sofisparkle chimed in, and if I remember correctly we may have even come to a conclusion on what to call each of these.

Reply:
In fact, it's the thread linked from https://github.com/sparkletown/sparkle/pull/1427#discussion_r638451622

Reply:
Is renamed in https://github.com/sparkletown/sparkle/pull/1427/commits/2ce258fde4bbcf2151b14086f53390bb874f2910